tensorflow/lite/kernels/reduce_test.cc

```diff
@@ -1221,5 +1250,89 @@ TEST(DynamicAnyOpTest, Scalar) {
   EXPECT_THAT(m.GetOutput<bool>(), ElementsAreArray({false}));
 }
 
+// Tests for reduce_all
+TEST(ConstAllOpTestFalse, NotKeepDims) {
```

Comment:
This test is fairly duplicative of the next test. Can you rename to ConstAllOpsTestMixed and use this data:

```
std::vector<bool> data = {true, true, false, false, true, true,
                            true, true,  false, false, true, true};
```

with an axis of `[0,2]`. The result will be `true, false, true`. (the extraneous `-3, -3` axis dimensions are specified in the next test so we are still testing that functionality.

Reply:
Also please add another version of the test described in the above comment with `keep_dims=true`. Thanks! Nice PR!